platform/darwin/test/MGLStyleTests.mm

```diff
@@ -28,7 +28,7 @@ - (void)setUp {
     [super setUp];
 
     [MGLAccountManager setAccessToken:@"pk.feedcafedeadbeefbadebede"];
-    NSURL *styleURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"one-liner" withExtension:@"json"];
+    NSURL *styleURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"basic-style" withExtension:@"json"];
```

Comment:
Unlike one-liner.json, basic-style.json seems to have network dependencies. We’d need to mock those dependencies to ensure stable tests.

Reply:
One possibility - we could (for the time being) convert this to an integration test, which do hit the network...